crates/fluvio-hub-util/src/packagemeta.rs

```diff
@@ -310,6 +344,18 @@ fn builds_obj_key_from_package_name() {
     }
 }
 
+// this end up mostly for any cli tools printing out the status
+impl std::fmt::Display for PkgVisibility {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::result::Result<(), std::fmt::Error> {
+        let lbl = match self {
+            PkgVisibility::Private => "private",
+            PkgVisibility::Public => "public",
+        };
+        write!(f, "{}", lbl)?;
+        Ok(())
```

Comment:
I think we can save ourselves the `Ok(())` and the `?` here by taking advantage of `write!` return types.

```suggestion
        write!(f, "{}", lbl)
```